lib/utils.js

```diff
@@ -57,23 +58,32 @@ async function getAndCheckIosSdkVersion () {
   }
 }
 
-function translateDeviceName (platformVersion, devName = '') {
-  let deviceName = devName;
-  switch (devName.toLowerCase().trim()) {
-    case 'iphone simulator':
-      deviceName = 'iPhone 6';
-      break;
-    case 'ipad simulator':
-      // iPad Retina is no longer available for ios 10.3
-      //   so we pick another iPad to use as default
-      deviceName = platformVersion && util.compareVersions(platformVersion, '<', '10.3') ? 'iPad Retina' : 'iPad Air';
+/**
+ * Get the generic simulator for a given IOS version and device type (iPhone, iPad)
+ *
+ * @param {string|number} platformVersion IOS version. e.g.) 13.0
+ * @param {string} deviceName Type of IOS device. Can be iPhone, iPad (possibly more in the future)
+ */
+function getGenericSimulatorForIosVersion (platformVersion, deviceName) {
+  const genericSimulators = iosGenericSimulators[deviceName];
+  let genericIosSimulator;
+
+  // Find the highest iOS version in the list that is below the provided version
+  for (const [platformVersionFromList, iosSimulator] of genericSimulators) {
+    if (util.compareVersions(platformVersionFromList, '>', platformVersion)) {
       break;
+    }
+    genericIosSimulator = iosSimulator;
   }
+  return genericIosSimulator;
+}
 
-  if (deviceName !== devName) {
-    log.debug(`Changing deviceName from '${devName}' to '${deviceName}'`);
+function translateDeviceName (platformVersion, deviceName = '') {
+  const translatedDeviceName = getGenericSimulatorForIosVersion(platformVersion, deviceName.toLowerCase().trim());
```

Comment:
I would opt for a variable name that is not so close to the function name, for readability's sake.